### PR TITLE
rosauth: 0.1.7-1 in 'lunar/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by rctoris**
Increasing version of package(s) in repository `rosauth` to `0.1.7-1`:

- upstream repository: https://github.com/GT-RAIL/rosauth.git
- release repository: https://github.com/gt-rail-release/rosauth-release.git
- distro file: `lunar/distribution.yaml`
- bloom version: `0.5.25`
- previous version for package: `null`

### CODE REVIEW

**Comment by tfoote** [comment]
The lunar release tags do not appear to be in the repo.

```
rosdistro: DEBUG: Skipped "github_manifest_provider()": specified tag "release/lunar/rosauth/0.1.7-1" is not a git tag
rosdistro: DEBUG: Skip non-bitbucket url "https://github.com/gt-rail-release/rosauth-release.git"
rosdistro: DEBUG: Skipped "bitbucket_manifest_provider()": Cannot handle non bitbucket url.
rosdistro: DEBUG: Skipped "git_manifest_provider()": Unable to fetch package.xml: Could not clone repository "https://github.com/gt-rail-release/rosauth-release.git" at reference "release/lunar/rosauth/0.1.7-1"
```

**Comment by rctoris** [comment]
hmmmmmm okay let me try again

**Comment by rctoris** [comment]
closing this out
